yabgp/common/constants.py

```diff
@@ -256,6 +256,32 @@
     'ORIGIN': 1,
     'ORIGINATOR_ID': 9}
 
+WELL_KNOW_COMMUNITY_STR_2_INT = {
+    'PLANNED_SHUT': 0xFFFF0000,
+    'ACCEPT_OWN': 0xFFFF0001,
+    'ROUTE_FILTER_TRANSLATED_v4': 0xFFFF0002,
+    'ROUTE_FILTER_v4': 0xFFFF0003,
+    'ROUTE_FILTER_TRANSLATED_v6': 0xFFFF0004,
+    'ROUTE_FILTER_v6': 0xFFFF0005,
+    'NO_EXPORT': 0xFFFFFF01,
+    'NO_ADVERTISE': 0xFFFFFF02,
+    'NO_EXPORT_SUBCONFED': 0xFFFFFF03,
+    'NOPEER': 0xFFFFFF04
+}
+
+WELL_KNOW_COMMUNITY_INT_2_STR = {
```

Comment:
WELL_KNOW_COMMUNITY_INT_2_STR =  = dict([(r,l) for (l,r) in WELL_KNOW_COMMUNITY_STR_2_INT.items()])